pom.xml

```diff
@@ -168,6 +168,7 @@
         <dom4j.bundle.version>1.6.1_5</dom4j.bundle.version>
         <eclipselink.version>2.7.1</eclipselink.version>
         <jasypt.bundle.version>1.9.2_1</jasypt.bundle.version>
+        <jolokia.version>1.5.0</jolokia.version>
         <jolokia.version>1.3.5</jolokia.version>
```

Comment:
This property should be removed to keep only the 1.5.0. By the way, did you test the new Jolokia release ?